### PR TITLE
Upload private docs

### PR DESCRIPTION
**Pull request by barchen1**
## Status
Ready

## Related Issues
related: https://github.com/demisto/etc/issues/30289

## Description
Added support for private packs docs

## Screenshots

Screen Shot 2021-02-11 at 16 25 18

### CODE REVIEW

File: content-repo/gendocs_test.py [hunk 1]
```diff
@@ -231,6 +231,20 @@ def test_process_extra_doc(tmp_path, mdx_server):
         assert f.readline().startswith('custom_edit_url: https://github.com/demisto/content-docs/blob/master/content-repo/extra-docs/integrations')
 
 
+def test_process_private_doc(tmp_path, mdx_server):
+    release_file = f'{os.path.dirname(os.path.abspath(__file__))}/.content-bucket/Packs/HelloWorldPremium/Playbooks/' \
```
Comment (glicht) [inline content-repo/gendocs_test.py:235]: Don't assume this directory exists. The test data you want to test should reside in `content-repo/test_data`.

**Review by glicht — COMMENTED** [review]
Looks good. Need to just fix the unit tests.

**Review by glicht — APPROVED** [review]
Excellent. Looks good.

**Review by glicht — COMMENTED** [review]
@barchen1 I am looking at the preview site and not seeing the following script documentation: Code42UsernameSearch

**Comment by glicht** [comment]
@barchen1 also look at the build. We have some failures for playbooks. Such as:

```
11:48:54.629980 fail: /home/circleci/project/content-repo/.content-bucket/Packs/SafeBreach_Insights/Playbooks/SafeBreach_Process_Behavioral_Insights_Feed_README.md. Exception: Traceback (most recent call last):
  File "/home/circleci/project/content-repo/./gendocs.py", line 319, in process_extra_readme_doc
    yml_data = yaml.safe_load(yml_matter)
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/__init__.py", line 162, in safe_load
    return load(stream, SafeLoader)
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/__init__.py", line 114, in load
    return loader.get_single_data()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/scanner.py", line 173, in fetch_more_tokens
    return self.fetch_stream_end()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/scanner.py", line 377, in fetch_stream_end
    self.remove_possible_simple_key()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/scanner.py", line 318, in remove_possible_simple_key
    raise ScannerError("while scanning a simple key", key.mark,
yaml.scanner.ScannerError: while scanning a simple key
  in "<unicode string>", line 3, column 1:
    A special feed based triggered j ... 
    ^
could not find expected ':'
  in "<unicode string>", line 3, column 119:
     ...  SafeBreach generated indicator.
                                         ^

11:48:54.738093 fail: /home/circleci/project/content-repo/.content-bucket/Packs/IAM/Playbooks/IAM_-_Check_If_User_Should_Be_Terminated_README.md. Exception: Traceback (most recent call last):
  File "/home/circleci/project/content-repo/./gendocs.py", line 319, in process_extra_readme_doc
    yml_data = yaml.safe_load(yml_matter)
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/__init__.py", line 162, in safe_load
    return load(stream, SafeLoader)
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/__init__.py", line 114, in load
    return loader.get_single_data()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/constructor.py", line 49, in get_single_data
    node = self.get_single_node()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/composer.py", line 36, in get_single_node
    document = self.compose_document()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/composer.py", line 55, in compose_document
    node = self.compose_node(None, None)
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/composer.py", line 84, in compose_node
    node = self.compose_mapping_node(anchor)
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/composer.py", line 127, in compose_mapping_node
    while not self.check_event(MappingEndEvent):
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/parser.py", line 98, in check_event
    self.current_event = self.state()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/parser.py", line 428, in parse_block_mapping_key
    if self.check_token(KeyToken):
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/scanner.py", line 116, in check_token
    self.fetch_more_tokens()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/scanner.py", line 223, in fetch_more_tokens
    return self.fetch_value()
  File "/home/circleci/.local/share/virtualenvs/project-zxI9dQ-Q/lib/python3.9/site-packages/yaml/scanner.py", line 577, in fetch_value
    raise ScannerError(None, None,
yaml.scanner.ScannerError: mapping values are not allowed here
  in "<unicode string>", line 2, column 169:
     ... wing fields to make its decision: Employee Status, Rehired Emplo ... 
                                         ^
```

**Comment by barchen1** [comment]
> @barchen1 I am looking at the preview site and not seeing the following script documentation: Code42UsernameSearch

Added a fix for private packs Scripts, it should work now

**Review by glicht — APPROVED** [review]

**Comment by glicht** [comment]
@barchen1 I think you can merge

**Comment by barchen1** [comment]
@glicht I want to merge this [PR](https://github.com/demisto/content-private/pull/199) first, cause I want to test it one more time with the description field fix I added in the `upload_docs.py`.

**Pull request merged**